flocker/provision/_install.py

```diff
@@ -1532,6 +1581,90 @@ def configure_cluster(
     ])
 
 
+def reinstall_flocker_at_version(
+    reactor, nodes, control_node, package_source, distribution
+):
```

Comment:
:question: Could this be shared with the ManagedRunner which also upgrades Flocker?